shell/platform/android/test/io/flutter/plugin/editing/TextInputPluginTest.java

```diff
@@ -1860,6 +1879,9 @@ public void autofill_doesNotCrashAfterClearClientCall() {
         .updateEditingState(anyInt(), any(), anyInt(), anyInt(), anyInt(), anyInt());
   }
 
+  @Config(minSdk = Build.VERSION_CODES.O)
+  @SuppressWarnings("deprecation")
+  // Robolectric.setupActivity.
```

Comment:
Helper for all of these in this file, to consolidate the deprecation warning and the TODO?

Reply:
Good idea. Migrated to test method for the class.